app/templates/main/index.html

```diff
@@ -3,13 +3,24 @@
 {% block content %}
     <link rel="stylesheet" type="text/css" href="static/styles/map.css">
     <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyAhkKJDnPxq0_xvF6pLAFcKfVAL6GbARBs&libraries=places&callback=initMap"
+        async defer></script>    
```

Comment:
I think the current position of `async` is actually stylistically "correct" for HTML (which doesn't really have style). I do think bumping `</script>` to the next line is a good idea though.